my_list.rb

```diff
@@ -7,6 +7,10 @@ def initialize(*arr)
   end
 end
 
+def each(&block)
+  @list.each(&block)
+end
```

Comment:
[OPTIONAL] I think you could try to implement a method `#each` using other utilities other than the `#each` from Ruby. This could give you a good insight into how `each` actually works.